Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.12 AS builder
+WORKDIR /go/src/github.com/openshift/kubernetes-nmstate
+COPY . .
+
+RUN GO111MODULE=on go build --mod=vendor -o kubernetes-nmstate cmd/manager/main.go
```

Comment:
If you want to use the same image for both operator, extend the Dockerfile in build/ instead of adding this duplicated one.

If you want to have dedicated Dockerfile for OpenShift, add it to build/Dockerfile.openshift. However, make sure that the basic Dockerfile we currently have is in sync with it.

Overall, I think we should start by sticking to the existing kubernetes-first approach and only once we have something functional merged, we can think how to plug it to OpenShift.